darts/models/forecasting/nbeats.py

```diff
@@ -656,6 +701,11 @@ def __init__(
         self.expansion_coefficient_dim = expansion_coefficient_dim
         self.trend_polynomial_degree = trend_polynomial_degree
 
+        # Currently batch norm is not an option as it seems to perform badly
+        self.batch_norm = False
```

Comment:
Don't you need to receive this as an argument?

Reply:
I wasn't sure if I should have included it or not. Enabling batch_norm performs very poorly on both NHiTs and NBEATs

I chose to matches the NHiTs implementation, but can remove it or add it as an arg.
https://github.com/unit8co/darts/blob/5213854b71e3d56d41dcdffac59140a8da9c10a7/darts/models/forecasting/nhits.py#L636-L637

Reply:
OK fair enough, let's leave it like that for now then.